pandas/tests/extension/base/ops.py

```diff
@@ -154,23 +153,22 @@ def _compare_other(self, ser: pd.Series, data, op_name: str, other):
                 with pytest.raises(type(exc)):
                     ser.combine(other, op)
 
-    def test_compare_scalar(self, data, all_compare_operators):
-        op_name = all_compare_operators
+    def test_compare_scalar(self, data, comparison_op):
         ser = pd.Series(data)
-        self._compare_other(ser, data, op_name, 0)
+        self._compare_other(ser, data, comparison_op, 0)
 
-    def test_compare_array(self, data, all_compare_operators):
-        op_name = all_compare_operators
+    def test_compare_array(self, data, comparison_op):
         ser = pd.Series(data)
         other = pd.Series([data[0]] * len(data))
-        self._compare_other(ser, data, op_name, other)
+        self._compare_other(ser, data, comparison_op, other)
 
-    @pytest.mark.parametrize("box", [pd.Series, pd.DataFrame])
-    def test_direct_arith_with_ndframe_returns_not_implemented(self, data, box):
+    def test_direct_arith_with_ndframe_returns_not_implemented(
+        self, data, frame_or_series
```

Comment:
Those base extension tests are used by downstream projects, so we should be a bit more careful with changing them (at least when for "clean up" purposes). Or use fixtures defined in `tests/extension/conftest.py` as you did in https://github.com/pandas-dev/pandas/pull/44138. 
I have changed back this specific line in https://github.com/pandas-dev/pandas/pull/44332